app/models/info_request.rb

```diff
@@ -888,11 +888,7 @@ def legislation
   end
 
   def law_used_human(key = :full)
-    begin
-      applicable_law.fetch(key)
-    rescue KeyError
-      raise "Unknown key '#{key}' for '#{law_used}'"
```

Comment:
We've lost this behaviour:

```ruby
r = InfoRequest.new(law_used: 'unknown')
r.law_used_human(:full)
# => "Freedom of Information"
```

I think an error is the correct thing to do when an invalid `law_used` key is provided.

Reply:
Restored in 3673650e49